### DISCUSSION TITLE
RFC9242

### DISCUSSION THREAD

**Discussion by keskinadem** [strongswan/strongswan#2181] [General]
I read something like "**For the purpose of calculating the IntAuth_[i/r]* values, the content of the IKE_INTERMEDIATE messages is represented as two chunks of data: mandatory IntAuth_[i/r]*A, optionally followed by IntAuth_[i/r]*P.**" in RFC9242.  Is IntAuth_[i/r]*P really optional in strongswan implementation? İf yes is there any configuration parameter related with this.

**Comment by tobiasbrunner** [comment]
You might want to read the RFC fully to learn what `IntAUth_[i/r]*P` is.
